README.md

```diff
@@ -66,10 +68,48 @@ Clicking the button will now toggle it's aria-expanded state. CSS can be used to
 }
 ```
 
+## Example 2: Requires aria-expanded and a class
+
+In this second example, our expanded content is not adjacent to the host element.
+
+```html
+<div class="expander">
+    <span>
+        <input class="expander__host" type="text">
+    </span>
+    <div class="expander__content">
+        <p>Any kind of HTML control can go inside...</p>
+        <p>A link: <a id="foo" href="http://www.ebay.com">www.ebay.com</a></p>
+        <p>A button: <button>Click Me</button></p>
+        <p>An input: <input type="text" aria-label="Dummy textbox"></p>
+        <p>A checkbox: <input type="checkbox" aria-label="Dummy checkbox"></p>
+    </div>
+</div>
+```
+
+A CSS classname is required as our styling hook. This can be passed in via the options.
+
+```js
+// options
+const options = {
+    expandedClass: 'expander--expanded',
+    expandOnFocus: true
+};
+```
+
+Setting focus on the host (a text input) sets it's aria-expanded state *and* add adds the chosen class to the root. CSS can be used to display the content accordingly, for example:
+
+```css
+.expander--expanded .expander__content {
+    display: block;
+}
+```
+
 ## Params
 
 * `el`: the root widget el
-* `options.alwaysDoFocusManagement`: honour `focusManagement` setting even when non-keyboard device is used (default: false)
+* `options.alwaysDoFocusManagement`: whether `focusManagement` option (see below) should apply for mouse click
+* `options.ariaControls`: specify whether `aria-controls` relationship should be created between host and overlay (default: true)
```

Comment:
Why would there not be a relationship when using the expander?

Reply:
Depending on the type of widget (menu, fake-menu, listbox, combobox, infotip, etc), aria-controls isn't always needed. In fact, some a11y checkers (axe, for example) are flagging it as an error. So I may need to remove aria-controls in one or two of our components. I need to dig into that a little more when I get time.